File: 1-HelloWorldAndLoops/helloWorld.py
print("Hello World!")
"""
someNumbers = [1,2,3,4,5]
print(someNumbers, len(someNumbers)) # [1,2,3,4,5] 5
"""
shoppingCart = []

item = ""
while item != "DONE":
    item = input("What do you want to add to your shopping cart?")
    shoppingCart.append(item)

for item in shoppingCart:
    print(item)

File: 1-HelloWorldAndLoops/helloWorld.swift
print("Hello world!")

//"Static typing"
var shoppingCart = [String]()
var item = ""
while (shoppingCart.count < 5 && item != "Done"){
    // get the item from the user
    print("What do you want to add to your cart?")
    if let userInput = readLine(){
        // add the item to the shopping cart
        item = userInput
        if (item != "Done"){
            shoppingCart.append(item)
        }
    }
}

// print everything in the cart to the screen
print(shoppingCart)